metadatastore/commands.py

```diff
@@ -548,25 +475,29 @@ def find(data=True, limit=50, **kwargs):
 
     Returns
     -------
-    result: metadatastore.document.Document
-        Returns RunStart objects
-        One can access events for this given RunStart as:
-        run_start_object.events
+    events : list
+        List of metadatastore.document.Document
     """
-    raise NotImplementedError()
-    br_objects = find_run_start(limit, **kwargs)
+    results = list()
+    try:
+        run_start_id = kwargs.pop('run_start').id
+        event_descriptors = EventDescriptor.objects(__raw__=run_start_id)
+    except KeyError:
+        pass
+    if event_descriptors:
+        for e_desc in event_descriptors.order_by('-time'):
+            query_dict = dict()
+            query_dict['descriptor_id'] = event_descriptor.id
+            query_dict.update(kwargs)
```

Comment:
There is a degeneracy, if you pass in both `run_start` and `descriptor_id`.  As written, the kwarg will always win and you will get back a bunch of copies of the events rather than all of the events.  Can't this all be done in one mongo call?  http://docs.mongodb.org/manual/reference/operator/query/in/#op._S_in